frontend/src/auth/constants.ts

```diff
@@ -1 +1 @@
-export const USER_ROLES_CLAIM = "user_roles"
+export const USER_ROLES_CLAIM = 'user_roles';
```

Comment:
(question, non-blocking): Do we really want to enforce single quotes vs double? Do we really want to enforce colons?

Reply:
They are both defaults Prettier options. Personally I would keep the single quotes and drop the semicolon :)